addon/models/choice.js

```diff
@@ -18,6 +18,10 @@ export class SingleChoice extends Choice {
     return selectedOption ? selectedOption.value : null;
   }
 
+  valueOf() {
+    return this.selection;
+  }
+
```

Comment:
Since the actual choice model is the unboxed value, I don't think we want to mess with its valueOf() method.

I.e. I think the purpose of this issue is to make sure that when we have the "boxed" value. i.e. the thing that has the actions, and that also could be morphed into a completely separate object in the case of Boolean `false` being boxed in an empty array `[]`

In the case of `choice` it _is_ the thing being boxed.